<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FlightSimulator</class>
 <widget class="QDialog" name="FlightSimulator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>UDP Client</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <property name="leftMargin">
        <number>7</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>7</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>16777215</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="sizeConstraint">
             <enum>QLayout::SizeConstraint::SetFixedSize</enum>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>IP Address:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="le_IpAddr">
                <property name="text">
                 <string>127.0.0.1</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Port Number:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="le_portNum">
                <property name="text">
                 <string>12345</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QPushButton" name="pb_StartListening">
                <property name="text">
                 <string>Start Listening</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pb_DisconnectUDP">
                <property name="text">
                 <string>Dissconnect</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QCheckBox" name="cb_simulateTrack">
                <property name="text">
                 <string>Simulate track when data is received</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QPushButton" name="pb_SimulateAllTracks">
                  <property name="text">
                   <string>Simulate Tracks</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pb_Stop">
                  <property name="text">
                   <string>Stop</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pb_Replay">
                  <property name="text">
                   <string>Replay</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pb_SaveData">
                  <property name="text">
                   <string>Save Data</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="messageDispalay"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
